docs/examples/longhorn.md

```diff
@@ -0,0 +1,134 @@
+<!--
+SPDX-FileCopyrightText: 2025 k0s authors
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Longhorn
+
+This tutorial covers installing Longhorn as a Helm extension on k0s. Longhorn provides distributed block storage for Kubernetes with features like snapshots, backup, and live upgrades.
+
+## Prerequisites
+
+- Install `open-iscsi` on every Linux node (required by Longhorn).
+- Ensure the `iscsid` service is enabled and running on each node.
+- Make sure nodes have a stable hostname and reachable IPs.
+
+Ubuntu/Debian:
+
+```bash
+sudo apt-get update
+```
+
+```bash
+sudo apt-get install -y open-iscsi
+```
+
+RHEL/CentOS/Rocky/Alma:
+
+```bash
+sudo yum install -y iscsi-initiator-utils
+```
+
+SUSE:
+
+```bash
+sudo zypper install -y open-iscsi
+```
+
+## Install via k0s Helm extension
+
+Add Longhorn as a Helm chart in your k0s ClusterConfig. Apply this configuration through your normal k0s configuration workflow.
```

Comment:
Is there some link to the upstream chart docs that we could include for reference?